pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -223,15 +223,16 @@ func (b *OFBridge) createGroupWithType(id GroupIDType, groupType ofctrl.GroupTyp
 	return g
 }
 
-func (b *OFBridge) DeleteGroup(id GroupIDType) bool {
-	g := b.ofSwitch.GetGroup(uint32(id))
-	if g == nil {
-		return true
+func (b *OFBridge) DeleteGroup(id GroupIDType) error {
+	ofctrlGroup := b.ofSwitch.GetGroup(uint32(id))
+	if ofctrlGroup == nil {
+		return nil
 	}
+	g := &ofGroup{bridge: b, ofctrl: ofctrlGroup}
 	if err := g.Delete(); err != nil {
-		return false
+		return fmt.Errorf("failed to delete the group in bundle: %w", err)
```

Comment:
The context doesn't have anything related to bundle. I think it should just say "failed to delete the group: %w". If `g.Delete` fails because of something related to bundle, the returned err from it should contains it.

Reply:
Thanks, updated.